test/test.dart

```diff
@@ -3,14 +3,16 @@ import "package:test/test.dart";
 import "package:objectbox/objectbox.dart";
 part "test.g.dart";
 
-@Entity(id: 1, uid: 1)
+@Entity()
 class TestEntity {
-  @Id(id: 1, uid: 1001)
+  @Id()
   int id;
 
-  @Property(id: 2, uid: 1002)
   String text;
 
+  @Property(type: OBXPropertyType.Short)
```

Comment:
We don't really need custom type sizes at the moment so let's not complicate things with those now (e.g. validation whether the value fits into short). Just map the built-in types for now

Reply:
Done.